Proteomics/ProteomicsExtenstionMethods.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Proteomics
+{
+    public static class ProteomicsExtenstionMethods
+    {
+        public static List<int> AllIndexesOf(this string str, string value)
```

Comment:
Can you add a comment for when this method is needed? Just something explaining why you wrote it.